### DISCUSSION TITLE
Disable macOS 1px window frame top

### DISCUSSION THREAD

**Discussion by oschrenk** [wezterm/wezterm#4684] [General]
I prefer maximimzed windows over full screen windows, and one of the thing that "annoys" me is that in a maximized window without decorations I can still see a 1px border at the top of the window

See

<img width="402" alt="Screenshot 2023-12-14 at 16 53 32" src="https://github.com/wez/wezterm/assets/52898/e2d937f5-af0e-4e24-9849-febfa1502324">

is there some way to turn this off?

My config:

```
config.window_decorations = "RESIZE"
config.enable_tab_bar = false

config.window_frame = {
	inactive_titlebar_bg = "#000000",
	active_titlebar_bg = "#000000",
	inactive_titlebar_fg = "#000000",
	active_titlebar_fg = "#000000",
	inactive_titlebar_border_bottom = "#000000",
	active_titlebar_border_bottom = "#000000",
}

config.window_frame = {
	border_left_width = "0",
	border_right_width = "0",
	border_bottom_height = "0",
	border_top_height = "0",
	border_left_color = "black",
	border_right_color = "black",
	border_bottom_color = "black",
	border_top_color = "black",
}

```

Maybe it relates to https://github.com/wez/wezterm/issues/2182 ?

**Comment by aveaves** [comment]
Did you figure out how to remove it? I'm having the same problem.

**Comment by oschrenk** [comment]
Unfortunately not. The only terminal I found that does not have that is alacritty, so I am using that instead. Looks better in my setup.On 31. Mar 2024, at 10:31, Aves ***@***.***> wrote:﻿
Did you figure out how to remove it? I'm having the same problem.

**Comment by teddygroves** [comment]
I think you can achieve this by adding the line `config.use_fancy_tab_bar = false`
